alpaca/entities.go

```diff
@@ -362,6 +363,11 @@ type APIError struct {
 }
 
 func APIErrorFromResponse(resp *http.Response) error {
+	if resp == nil {
+		return errors.New("response is response")
+	}
+	defer resp.Body.Close()
```

Comment:
If you check all references of the `APIErrorFromResponse()` function, `resp.Body.Close()` is called. E.g. in `alpaca/rest.go:1334`. Moving the call here makes sense, but then those calls should be removed.